### ISSUE TITLE
Artwork

### ISSUE THREAD

**Issue by TheMrIron2** [darkhaven3/projectFrost#5]
Discussion regarding artwork to replace the generic Insomnia ProQuake logo and background.

Reminder of dimensions: 134x74 icon, 480x272 background.

**Comment by darkhaven3** [comment]
Not a bug

**Comment by TheMrIron2** [comment]
Well spotted

Issues can be kept open for things that are todo/in progress for something more than a basic project card

**Comment by darkhaven3** [comment]
Yes but that isn't what Github issues are designed for, that's why this issue is still open with its own label
